lib/evss/evss_common/configuration.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+module EVSS
+  module EVSSCommon
```

Comment:
I'm sure I don't have all the background context, but is there a reason this couldn't be `EVSS::Common` rather than `EVSS::EVSSCommon`? I see there are other namespaces like `EVSS::Claims` and `EVSS::Docuemnts` which is why I ask.

Reply:
i originally had named it `EVSS::Common` but in order to do that i have to change all references to the other `Common` in the EVSS namespace to `::Common` and i figured it was easier to just name this `EVSSCommon`

Reply:
Makes sense. Thanks!